en/publish/publish-wechatgame.md

```diff
@@ -1,114 +1,99 @@
-# 发布到微信小游戏平台
+# Publish to __WeChat__ Mini Game Platform
 
-早在微信官方正式发布小游戏平台之前，Cocos Creator 团队就和微信开发团队合作完成了 Cocos Creator 对微信小游戏平台的支持。在小游戏正式公布之后，我们也在第一时间发布了完美支持小游戏的 Cocos Creator v1.8。这篇文档的初衷也是希望帮助开发者们在第一时间以最顺畅的方式提交游戏到小游戏平台。
+![](./publish-__WeChat__game/preview.jpeg)
 
-![](./publish-wechatgame/preview.jpeg)
-
-这篇文档将会覆盖：
```

Comment:
Why did you delete this part of the front?

Reply:
We don’t say that. It’s redundant.

Reply:
OK~~